devices/philips.js

```diff
@@ -1067,6 +1067,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LCW002', '4090231P9'],
+        model: '4090231P9',
+        vendor: 'Philips',
+        description: 'Hue Liane',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Reply:
![image](https://user-images.githubusercontent.com/4787591/142662418-d235a6af-c78f-44e8-a775-db0678009a93.png)